### ISSUE TITLE
Cargolifter Incompatible With Mail 13.0

### ISSUE THREAD

**Issue by jamespereira62** [Mailbutler/cargolifter-mail-plugin#6]
I just upgraded to OS Catalina and Cargolifter is incompatible with Mail 13.0.

Is there a solution to this?

**Comment by jackl2400** [comment]
Yup, same problem. Unfortunately, Mailbutler support THINKS their latest version (53) works, but for many of us, it doesn't.

**Comment by wtowbin** [comment]
Same here ....  The latest Cargolifter DOES NOT work with Catalina. Attachments are not being uploaded separately to relevant server (Dropbox, etc).  
Please let me know how can I help in troubleshooting (wireshark, debug images, logs ..).

Thx, Walter

**Comment by fjaeger** [comment]
Could you please try with the latest version from here: https://github.com/Mailbutler/cargolifter-mail-plugin/releases

**Comment by 3ec** [comment]
Very good. This one seems to work.

Test anbei.

Best,
Frank

 
 <applewebdata://0A763DD9-B210-4143-BB65-C1120E872E30#>	
Frank Weber	
a: Rudolf-Müller-Str. 38, 36364 Bad Salzschlirf t: +49 6648 917151 <tel:+49 6648 917151>	
e: frank.weber@gmail.com <mailto:frank.weber@gmail.com>	
 
 <https://beta.mailbutler.io/tracking/hit/6697dc24-7387-4b13-a89d-dd00392b9e6f/db66d80a-6f4a-409d-9cd6-bc36dcde4d41>  <skype:fwcc-skype?call>

**Comment by jackl2400** [comment]
Hi, Fabian:

Still not working for me. Installs OK. Check box and restart in Mail “preferences>manage plug ins” OK. Re-link Dropbox and reinstall OK. But whenever I check the CargoLifter box to upload a file, the mail won’t send. Only after the CargoLifter box is unchecked will the message be sent (with attachment).

Let me know if I can help you with sending any config or error files, etc.



J

**Comment by 3ec** [comment]
… oder doch nicht.

My installation of the new version (54) and 2 subsequent test mails showed that 
installing this Cargolifter version is not rejected by Apple any more
the expected checkbox shows up when sending a mail in Apple Mail, but
even when the Cargolifter checkbox is checked and the attachment should be routed over Google Drive,
Apple Mail just sends it as a normal attachment, not via Google Drive plus URL in the mail.

So a 9 MB attachment looks like this, even when "via Cargolifter" is checked AND the preferences are „automatically via Cargolifter when above 5 MB":




Sorry about this news.


Best,
Frank

**Comment by wtowbin** [comment]
Fabian, guys,

Ver 54 still does not work for me. Behaviour as before.

W
-----------------------------------------------------
 
W Towbin

Email:  wtowbin@gmail.com
Public PGP Key:  https://www.dropbox.com/s/x1ontdnwvzpanz4/Walter%20Towbin%20%288D5C7C1F%29%20%E2%80%93%20Public.asc?dl=0

**Comment by jamespereira62** [comment]
I can install, but when I send email with attachment (19 MB) still no Cargolifter checkbox to select it.
